plugins/inputs/execd/execd.go

```diff
@@ -85,17 +86,23 @@ func (e *Execd) Start(acc telegraf.Accumulator) error {
 		return err
 	}
 
+	e.startDoneWg.Add(2) // stream reader synchronization
```

Comment:
When last I remember looking, I couldn't figure out why we needed this waitgroup, can you explain?

Reply:
it gives the stdin/stdout stream readers, which are in separate goroutines, time to start before Stop is allowed to kill the context and start shutting everything down. ... buuut.. I think we reorganized things and it's no longer needed because we aren't trying to stop the plugin before reading the value... Tests seem to suggest I can remove it 👍